waspc/cli/src/Wasp/Cli/Command/CreateNewProject.hs

```diff
@@ -14,21 +20,26 @@ import qualified System.FilePath as FP
 import Text.Printf (printf)
 import Wasp.Analyzer.Parser (isValidWaspIdentifier)
 import Wasp.Cli.Command (Command, CommandError (..))
+import Wasp.Cli.Command.Message (cliSendMessageC)
 import qualified Wasp.Data as Data
+import qualified Wasp.Generator.Job as J
+import Wasp.Generator.Job.Process (runNodeCommandAsJob)
+import qualified Wasp.Message as Msg
 import Wasp.Project (WaspProjectDir)
-import Wasp.Util (indent, kebabToCamelCase)
+import Wasp.Util (indent, kebabToCamelCase, whenM)
 import qualified Wasp.Util.IO as IOUtil
 import qualified Wasp.Util.Terminal as Term
 import qualified Wasp.Version as WV
 
 data ProjectInfo = ProjectInfo
   { _projectName :: String,
-    _appName :: String
+    _appName :: String,
+    _template :: Maybe String
```

Comment:
I found it a tiny bit weird that you put template here, I imagined ProjectInfo to be about the names, but on the other hand maybe this is fine! I don't have super great ideas how to keep it outside so I am ok with this. @sodic maybe you have an opinion on this?

Reply:
I'll change it to `templateName` to be more inline with other info in there.